src/notifications/notificationmanager.cpp

```diff
@@ -1388,17 +1388,26 @@ void NotificationManager::execSQL(const QString &command, const QVariantList &ar
     m_databaseCommitTimer.start();
 }
 
-void NotificationManager::invokeAction(const QString &action)
+void NotificationManager::invokeAction(const QString &action, const QString &actionText)
 {
+    Q_UNUSED(actionText)
```

Comment:
Is this some leftover? AFAICT the argument is used in the code.

Reply:
Yes indeed.